src/main/java/org/apache/commons/lang3/reflect/TypeUtils.java

```diff
@@ -1427,6 +1438,7 @@ public static Type unrollVariables(Map<TypeVariable<?>, Type> typeArguments, fin
      * @return boolean
      * @since 3.2
      */
+    @SuppressWarnings("array.access.unsafe.high.constant") // getImplicitLowerBounds returns an array with minimum length 1
```

Comment:
Always? If so, then annotate it. Otherwise, tell what's different about this use.